src/main/java/org/gaul/s3proxy/Main.java

```diff
@@ -235,6 +246,15 @@ public static void main(String[] args) throws Exception {
         }
     }
 
+    private static PrintStream createLoggerErrorPrintStream() {
+        return new PrintStream(new LogOutputStream() {
+            @Override
+            protected void processLine(String s, int i) {
```

Comment:
@andrewgaul we can write but I should take care of all the scenarios. If I have to write something, it will be equivalent to that one at the end. Can I copy that file into s3proxy repo instead of adding the dependency? Is that ok?